src/Symfony/Component/Console/Application.php

```diff
@@ -41,8 +41,10 @@
 use Symfony\Component\Console\Output\ConsoleOutputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
-use Symfony\Component\Debug\ErrorHandler;
-use Symfony\Component\Debug\Exception\FatalThrowableError;
+use Symfony\Component\Debug\ErrorHandler as LegacyErrorHandler;
+use Symfony\Component\Debug\Exception\FatalThrowableError as LegacyFatalThrowableError;
+use Symfony\Component\ErrorHandler\ErrorHandler;
+use Symfony\Component\ErrorHandler\Exception\FatalThrowableError;
```

Comment:
ideally we would get rid of that one :)

Reply:
We will (as we discussed on Slack). But let's fix the ErrorHandler support here, before another PR that remove the "fatal throwable errors" behavior.